src/include/esp-idf/bindings.h

```diff
@@ -46,6 +46,10 @@
 #include "esp_tls.h"
 #endif
 
+#ifdef ESP_IDF_COMP_ESP_VFS_FAT_ENABLED
+#include "esp_vfs_fat.h"
+#endif
+
 #ifdef ESP_IDF_COMP_APP_UPDATE_ENABLED
```

Comment:
I don't think a component named "esp_vfs_fat" exists? Usually, the components are named after their corresponding names in the `components/` folder of ESP-IDF (modulo replacing - with _). I don't see `esp_vfs_fat` there. What _is_ there, is `fatfs`.